dlang.org.ddoc

```diff
@@ -27,10 +27,10 @@ _=
 
 COMMON_SCRIPTS =
     $(SCRIPTLOAD https://ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js)
-    $(SCRIPT window.jQuery || document.write('<script src="$(ROOT)/js/jquery-1.7.2.min.js">\x3C/script>'))
-    $(SCRIPTLOAD $(ROOT)/js/codemirror-compressed.js)
-    $(SCRIPTLOAD $(ROOT)/js/run.js)
-    $(SCRIPTLOAD $(ROOT)/js/cssmenu.js)
+    $(SCRIPT window.jQuery || document.write('<script src="$(ROOT_DIR)js/jquery-1.7.2.min.js">\x3C/script>'))
```

Comment:
Microsoft's CHM generator seems to do funny things with `./`, it imports files in the current folder if it doesn't find a file in the folder of the file it's reading, thus a missed link causes it to pull in non-processed HTML files (meant for dlang.org) as well as preprocessed HTML files meant for the CHM.